ineom/src/main/java/com/inepex/ineom/shared/validation/basicvalidators/EmailValidator.java

```diff
@@ -13,11 +13,11 @@
     private final ObjectDesc objectDesc;
 
     public final static String regExpr = "^" + "([a-zA-Z0-9]){1,40}" + // firstname
-        "(([_\\.\\-\\+])([a-zA-Z0-9]){1,40})*" + // .lastname or
+        "(([_.\\-+])([a-zA-Z0-9]){1,40})*" + // .lastname or
```

Comment:
I am not a regex expert, but is this for sure redundant?
Will the . in the part [_.\\-+] not mach any character?